### PR TITLE
Fix broken links detected by make linkcheck-files

### PR DESCRIPTION
**Pull request by melissawm**
# References and relevant issues
Related to
https://github.com/napari/docs/pull/474

# Description
Fixes a number of broken links detected by `make linkcheck-files`.

A few comments:
- it looks like this link does not exist anymore: `further-resources/sample_data.md:17: [redirected permanently] [digitalpathologyassociation.org/whole-slide-imaging-repository](https://digitalpathologyassociation.org/whole-slide-imaging-repository) to [https://digitalpathologyassociation.org?preview_error](https://digitalpathologyassociation.org/?preview_error)`. I tried googling but only found this: http://www.wsirepository.org/ and it doesn't load. Any ideas?
- How can we replace the following link? `developers/architecture/app_model.md:288: [broken] https://github.com/napari/napari/blob/main/napari/_app_model/_submenus.py: 404 Client Error: Not Found for url: https://github.com/napari/napari/blob/main/napari/_app_model/_submenus.py` I couldn't find SubmenuItem in the codebase, maybe I'm missing something
- https://github.com/potating-potato has probably deleted their github account. I suggest we convert the links to static text if we want to preserve history.
- These hackmd docs seem to only be readable if you are logged in: https://hackmd.io/fmKp0If5RkiwWIxYYRdKpg and https://hackmd.io/@talley/SJB_lObBi#What-is-a-virtual-environment Should we make it readable by all, or ignore in the linkcheck?
- A few links will need to be fixed in npe2, I'll send a follow-up PR for that.
- This file does not seem to exist anymore, from a search at the btrack repo: `tutorials/tracking/cell_tracking.md:229: [broken] https://github.com/quantumjot/btrack/blob/main/models/cell_config.json: 404 Client Error: Not Found for url: https://github.com/quantumjot/btrack/blob/main/models/cell_config.json`. Should we replace with a new link?
- I will add a regex to ignore redirects from docs paths to stuff like "stable/" or "en/latest"

A few items are left, mostly links to old code. I will try to replace with permalinks where I can, but not sure if I'll be able to find all of them. If that's the case I might try to ping the original authors for an appropriate substitution.

### CODE REVIEW

File: docs/howtos/layers/labels.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ layer:
 
   Use this tool to manually erase a label on the `labels layer`. Other layers
   will not be affected. The label eraser tool looks like this:
-  ![image: eraser tool](../../images/labels-layer-eraser.png)
+  ![image: eraser tool](../../images/layers-labels-eraser.png)
```
Comment (psobolewskiPhD) [inline docs/howtos/layers/labels.md]: This doesn't appear correct, see:
[https://github.com/napari/docs/blob/main/docs/images/labels-layer-eraser.png](https://github.com/napari/docs/blob/main/docs/images/labels-layer-eraser.png)

It's causing a warning:
https://github.com/napari/docs/actions/runs/10563443854/job/29263666686?pr=484#step:9:1436
Let's revert?
```suggestion
  ![image: eraser tool](../../images/labels-layer-eraser.png)
```

File: docs/community/team.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ You can read a full history of the project’s founding and early development in
 
 Over time, napari has grown to over 80 direct contributors. Talley Lambert, from Harvard Medical School, began contributing around the time of Juan's blog post, and has been a critical driver of napari's development ever since. We have core developers located in three continents and three biweekly community calls spread across the day to try to accommodate users and contributors in any time zone.
 
-Read more about napari’s [mission and values](https://napari.org/community/mission_and_values.html), how to get started as a [contributor](https://napari.org/developers/contributing.html) or join us in our [zulip chat](https://napari.zulipchat.com/login/) for a more synchronous conversation. You can also follow us on [twitter](https://twitter.com/napari_imaging).
+Read more about napari’s [mission and values](https://napari.org/community/mission_and_values.html), how to get started as a [contributor](https://napari.org/developers/contributing.html) or join us in our [zulip chat](https://napari.zulipchat.com/login/) for a more synchronous conversation. You can also follow us on [twitter](https://x.com/napari_imaging).
```
Comment (psobolewskiPhD) [inline docs/community/team.md]:
```suggestion
Read more about napari’s [mission and values](https://napari.org/stable/community/mission_and_values.html), how to get started as a [contributor](https://napari.org/stable/developers/contributing.html) or join us in our [zulip chat](https://napari.zulipchat.com/login/) for a more synchronous conversation. You can also follow us on [twitter](https://x.com/napari_imaging).
```
I think those links work but rely on redirects/symlinks we want to get rid of long term right?

File: docs/tutorials/processing/dask.md [hunk 3]
```diff
@@ -194,7 +194,7 @@ for NeuroImaging of Coma and Consciousness, which is dedicated to promoting
 recovery of consciousness in people with severe brain injuries. This dataset is
 a 100 micron resolution magnetic resonance imaging (MRI) scan of an ex vivo
 human brain specimen. The brain specimen was donated by a [58-year-old woman](https://www.youtube.com/watch?v=Q-9jzBkoNuI) who had no history of neurological disease and died of non-neurological causes.
-This dataset was originally described in [Edlow, B.L., Mareyam, A., Horn, A. _et al._ 7 Tesla MRI of the ex vivo human brain at 100 micron resolution. Sci Data 6, 244 (2019)](https://doi.org/10.1038/s41597-019-0254-8).
+This dataset was originally described in [Edlow, B.L., Mareyam, A., Horn, A. _et al._ 7 Tesla MRI of the ex vivo human brain at 100 micron resolution. Sci Data 6, 244 (2019)](https://www.nature.com/articles/s41597-019-0254-8).
```
Comment (psobolewskiPhD) [inline docs/tutorials/processing/dask.md]: Is this change necessary? using the DOI is the best-practice.

**Review by psobolewskiPhD — CHANGES_REQUESTED** [review]
Wow! Thanks for catching all those!
I caught one small issue and left a comment

File: docs/howtos/layers/labels.md (hunk 1 above)
Comment (melissawm) [reply docs/howtos/layers/labels.md]: Oh man, search and replace all strikes again 🙈 Will fix

File: docs/community/team.md (hunk 2 above)
Comment (melissawm) [reply docs/community/team.md]: Good catch, will include

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by psobolewskiPhD** [comment]
Thanks!!
I wonder if we can squeeze this into 0.5.4 if we merge quickly?
cc: @jni

**Comment by melissawm** [comment]
Hi all - any other suggestions here?

**Review by DragaDoncila — APPROVED** [review]
LGTM! Let's get this in. Thanks for the ping @melissawm

**Comment by psobolewskiPhD** [comment]
Sorry I dropped the ball on this one 😭 
Thanks for picking it up!
